sandbox/modules/identity/src/main/java/org/opensearch/identity/rest/Endpoint.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.rest;
+
+public enum Endpoint {
```

Comment:
Do you mind explaining how this is used?

Reply:
It's unused. Deleting it.